### PR TITLE
Add button subjects#157

### LINKED ISSUE #157: (SP: 2 ) Add button and chips to Subjects step
UserStory: #75
Blocked by: #156 

## Description
Add "Add one more subject" button and chips of selected subjects on Subjects step for a tutor stepper

## Design 
Image
## Acceptance Criteria
- [x] When user selects subject and clicks on button "Add one more subject" it should create a new chip with selected subject
- [x] When user have selects 3rd or more subject it should be showed by increasing values of rest selected subjects

### PR DESCRIPTION
**Pull request by Mygal-Alex**
closed #157
image
- [x] When user selects subject and clicks on button "Add one more subject" it should create a new chip with selected subject
- [x] When user have selects 3rd or more subject it should be showed by increasing values of rest selected subjects

<h1>I can't increase test coverage because of a MUI bug<h1/>

### CODE REVIEW

**Review by ArtemHolikov — APPROVED** [review]
👍

**Review by marianamartsy — APPROVED** [review]
👍

**Review by AndriiNasadiuk — APPROVED** [review]

**Review by LeonidShamarin — APPROVED** [review]

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx [hunk 1]
```diff
@@ -44,16 +48,33 @@ const SubjectsStep = ({ btnsBox }) => {
...
-
     fetchSubjects()
   }, [selectedCategory])
-
+  const dataChipList = {
+    items: [...selectedSubjects],
+    defaultQuantity: 2,
+    handleChipDelete: (deletedItem) =>
+      setSelectedSubjects(
+        [...selectedSubjects].filter((item) => item !== deletedItem)
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx]: `filter` returns a new array. We do not need `...` here.
```suggestion
        selectedSubjects.filter((item) => item !== deletedItem)
```

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx [hunk 2]
```diff
@@ -44,16 +48,33 @@ const SubjectsStep = ({ btnsBox }) => {
...
   const handleCategoryChange = (event, newValue) => {
     setSelectedCategory(newValue)
     setSelectedSubject(null)
   }
+  const addSubjects = () => {
+    setSelectedSubject(null)
+    setSelectedSubjectName(true)
+    if (selectedSubject != null) {
+      if (!selectedSubjects.includes(selectedSubject.name)) {
+        setSelectedSubjects([...selectedSubjects, selectedSubject?.name])
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx]: Do we need `?` here. We already have check on `null` on 69th row.

**Review by vlad-khrychov — APPROVED** [review]
What about creating a new task or Spike for improving test coverage. If you add it, please add notes about it in description "Spike for improving test coverage was added #number_of_new_issue".

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx (hunk 2 above)
Comment (Mygal-Alex) [reply src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx]: i fix it

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx (hunk 1 above)
Comment (Mygal-Alex) [reply src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx]: i fix it

**Pull request merged**